### PR TITLE
add health check page

### PR DESCRIPTION
**Pull request by sshelomentsev**

### CODE REVIEW

File: src/env.ts [hunk 1]
```diff
@@ -50,6 +50,10 @@ export function getHealthCheckPath(env: WorkerEnv) {
   return `/${getWorkerPathVar(env)}/health`
 }
 
+export function getHealthPagePath(env: WorkerEnv) {
```
Comment (necipallef) [inline src/env.ts]: I think we can get rid of the word "health" and use "status" instead. Let's call this function `getStatusPagePath`,

File: src/handler.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import { getScriptDownloadPath, getGetResultPath, getHealthCheckPath, WorkerEnv } from './env'
+import { getScriptDownloadPath, getGetResultPath, getHealthCheckPath, WorkerEnv, getHealthPagePath } from './env'
 
 import { createErrorResponse } from './utils'
-import { handleDownloadScript, handleIngressAPI, handleHealthCheck } from './handlers'
+import { handleDownloadScript, handleIngressAPI, handleHealthCheck, handleHealthPage } from './handlers'
```
Comment (necipallef) [inline src/handler.ts]: Let's call the file `handleStatusPage`

File: src/handlers/handleHealthPage.ts [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  const isWorkerPathAvailable = isWorkerPathSet(env)
+  const isScriptDownloadPathAvailable = isScriptDownloadPathSet(env)
+  const isGetResultPathAvailable = isGetResultPathSet(env)
+  const isAllVarsAvailable = isWorkerPathAvailable && isScriptDownloadPathAvailable && isGetResultPathAvailable
+
+  let result = ''
+  if (!isAllVarsAvailable) {
+    result += `
+    <span>
+    The folliwing environment variables are not defined. Please reach out our support team.
```
Comment (necipallef) [inline src/handlers/handleHealthPage.ts]: typo, folliwing -> following

**Review by necipallef — CHANGES_REQUESTED** [review]

File: src/handler.ts (hunk 2 above)
Comment (sshelomentsev) [reply src/handler.ts]: agree

File: src/env.ts (hunk 1 above)
Comment (sshelomentsev) [reply src/env.ts]: ok

File: src/handlers/handleHealthPage.ts (hunk 3 above)
Comment (sshelomentsev) [reply src/handlers/handleHealthPage.ts]: fixed

**Review by necipallef — APPROVED** [review]

**Pull request merged**